charts/monitoring-config/rules/email_alert_api_alerts_tests.yaml

```diff
@@ -0,0 +1,96 @@
+rule_files:
+  - email_alert_api_alerts.yaml
+
+evaluation_interval: 1m
+
+tests:
+  # Alert does not fire if current and delivered are identical
+  - name: travel_advice_alert_test_identical
+    interval: 1m
```

Comment:
These [default to evaluation_interval](https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/#test_group).
```suggestion
```

Reply:
Done